### DISCUSSION TITLE
useRefreshToken types error

### DISCUSSION THREAD

**Discussion by scr2em** [react-auth-kit/react-auth-kit#599] [Q&A] [answered]
why did you comment out those lines, they causes type errors in the latest version 

https://github.com/scr2em/react-auth-kit/commit/7a7a29d733315127f73257f1b9ae42a1438c55ab#diff-0e2616be6cb3ff4e180a804ecc1c64b3995b6f87860c05b45eed3dd44949930c

**Comment by darkmatter18** [comment, marked as answer]
Hey, I commented out those lines, because I wanted to change the refresh Token. 

I made PR #603 which has all the new code bases.

Now I am stuck. I need help. Actually, I want to test that codebase. But the problem is to test the code, I have to release it to NPM. So do you have any idea, how can I test the code without publishing.

**Comment by darkmatter18** [comment]
@tumit @atrievel Can you give me any suggestions?

**Reply by scr2em** [reply]
you have 2 options 
1- build it then install from local 
```
npm i -S < build directory > // npm i -S ./build
```

2- install from github directly 
```
npm i --save < userName >/< repoName >#< branchName >  // npm i react-auth-kit/react-auth-kit#refresh-token  --save
```

**Reply by darkmatter18** [reply]
Thanks a lot. ❤️
